src/view/components/extensionViewForm.jsx

```diff
@@ -0,0 +1,86 @@
+import React, { useEffect, useContext, useRef } from "react";
+import { useFormik, FormikProvider } from "formik";
+import PropTypes from "prop-types";
+import wrapValidateWithErrorLogging from "../utils/wrapValidateWithErrorLogging";
+import ExtensionViewContext from "./extensionViewContext";
+
+const ExtensionViewForm = ({
+  initialValues,
+  getSettings,
+  validate,
+  validationSchema,
+  render
```

Comment:
We talked about adding another prop here for validating things that aren't Formik-related. Did you want to do that? I can do it when I start to use this on my branch if you want.

Reply:
You are welcome to add that.